src/infill.cpp

```diff
@@ -4,9 +4,20 @@
 #include "utils/polygonUtils.h"
 #include "utils/logoutput.h"
 
-namespace cura {
+/*!
+ * Function which returns the scanline_idx for a given x coordinate
+ *
+ * For negative \p x this is different from simple division.
+ *
+ * \warning \p line_width is assumed to be positive
+ *
+ * \param x the point to get the scansegment index for
+ * \param line_width the width of the scan segments
+ */
+static inline int computeScanSegmentIdx(int x, int line_width);
```

Comment:
In my opinion if the function is used only in this class then is should be private and not visible from outside. It makes code more understandable, because if after a few months some body will review/use this code he will may assume that this function is also used somewhere else.